bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/util/exceptions/BallerinaErrorReasons.java

```diff
@@ -54,6 +55,7 @@
             "DecimalExponentError"));
     public static final BString ARITHMETIC_OPERATION_ERROR =
             StringUtils.fromString(BALLERINA_PREFIX.concat("ArithmeticOperationError"));
+    public static final BString INVALID_OPERATION_ERROR = getModulePrefixedReason(DECIMAL_LANG_LIB, "InvalidOperation");
```

Comment:
The name doesn't indicate it's a decimal langlib error.

Not consistent with other langlib generic errors, either? https://github.com/ballerina-platform/ballerina-lang/blob/f8013e1400236f1a44cfc33001706e5e283529c4/bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/util/exceptions/BallerinaErrorReasons.java#L96

Can either be 
```suggestion
    public static final BString DECIMAL_OPERATION_ERROR = getModulePrefixedReason(DECIMAL_LANG_LIB, "DecimalOperationError");
```

or specifically
```suggestion
    public static final BString QUANTIZE_ERROR = getModulePrefixedReason(DECIMAL_LANG_LIB, "QuantizeError");
```

I prefer the latter since this is only used in quantize.